src/tasks/installers/helm.ts

```diff
@@ -82,8 +112,24 @@ export class HelmTasks {
           }
         }
       },
+      {
+        title: 'Check Cluster Role Binding',
```

Comment:
Do we really need to remove the cluster role binding if it exist? If it's not mandatory I would not add this extra step because it may fail (for example because the user doesn't have enough privileges), it's something the user hasn't requested and it doesn't bring any value.

Reply:
Issue is that helm complain about existing role if I launch server start again if I remove che namespace

Reply:
Issue is that helm complain about existing role if I launch server start again if I remove che namespace